server/src/pages/shot/time-diff.js

```diff
@@ -24,6 +24,12 @@ exports.TimeDiff = class TimeDiff extends React.Component {
     }
   }
 
```

Comment:
We still need to deal with proper plural forms with "x days".
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals